src/vivarium_nih_us_cvd/components/treatment.py

```diff
@@ -644,6 +659,55 @@ def apply_ldlc_treatment_ramp(
             + 1
         ).map(self.ldlc_treatment_map)
 
+        # Apply outreach intervention to groups 'newly_prescribed' (d, e, f)
+        # and simulants already on medication
+        maybe_enroll = newly_prescribed.union(currently_medicated)
+        pop_visitors = self.enroll_in_outreach(pop_visitors, maybe_enroll)
+
+        return pop_visitors
+
+    def enroll_in_outreach(
+        self, pop_visitors: pd.DataFrame, maybe_enroll: pd.Index
+    ) -> pd.DataFrame:
+        """Enrolls simulants in outreach intervention programs. It updates
+        the outreach column as well as both medication adherence columns (the
+        effect of outreach intervention).
+
+        Reminder: 'cat1' outreach means enrolled and 'cat2' means not enrolled.
+        """
+        current_outreach = pop_visitors.loc[maybe_enroll, data_values.COLUMNS.OUTREACH]
+        new_outreach = self.outreach(maybe_enroll)
+        to_enroll = current_outreach[current_outreach != new_outreach].index
+        if not to_enroll.empty:
+            breakpoint()  # save for next PR
+            # Update the outreach column with the new pipeline values. This
+            # is then used in OutreachEffect which registers a value modifier
+            # using the (newly updated) outreach column to modify adherence exposure.
+            pop_visitors.loc[to_enroll, data_values.COLUMNS.OUTREACH] = new_outreach.loc[
+                to_enroll
+            ]
+            self.population_view.update(pop_visitors[[data_values.COLUMNS.OUTREACH]])
+            # With the just-updated outreach column, update the medication
+            # adherence columns with pipeline values
+            pop_visitors.loc[
+                to_enroll,
+                [
+                    data_values.COLUMNS.SBP_MEDICATION_ADHERENCE,
+                    data_values.COLUMNS.LDLC_MEDICATION_ADHERENCE,
+                ],
+            ] = [
+                self.sbp_medication_adherence(to_enroll),
+                self.ldlc_medication_adherence(to_enroll),
+            ]
+            self.population_view.update(
```

Comment:
Why are you updating these columns both here and in line 440?

Reply:
oversight. For my own clarification, though - it wouldn't change anything right?